apps/aechannel/src/aesc_slash_tx.erl

```diff
@@ -119,13 +118,14 @@ process(#channel_slash_tx{channel_id = ChannelId,
 -spec accounts(tx()) -> list(pubkey()).
 accounts(#channel_slash_tx{payload = Payload}) ->
     case deserialize_from_binary(Payload) of
-        {ok, SignedState, _StateTx} -> aetx:signers(aetx_sign:tx(SignedState));
+        {ok, SignedState, _StateTx} -> aetx:signers(aetx_sign:tx(SignedState),
```

Comment:
But `aetx:signers/2` returns `{ok, list[pubkey()]}`.
Doesn't Dialyzer complain about this?

Reply:
Great catch!